### ISSUE TITLE
Is fastspeech training slow?

### ISSUE THREAD

**Issue by DabiaoMa** [TensorSpeech/TensorFlowTTS#441]
![SSB00430433](https://user-images.githubusercontent.com/24405575/102987097-d0551500-454c-11eb-93a5-0ed47d6c585d.png)ave
![SSB01330002](https://user-images.githubusercontent.com/24405575/102987101-d0edab80-454c-11eb-8ac2-96bd01890201.png)

 I have trained a multi speaker (Aishell dataset) fastspeech model for 60K steps with batch_size 32, lr 1e-4. But the eval mel figs are still blurring, what is the expected behavior of fastspeech? The training loss has decreased to about 0.17, but it decays very slowly. 
![SSB00090464](https://user-images.githubusercontent.com/24405575/102987061-c3382600-454c-11eb-8403-e9c978f94d7c.png)

**Comment by dathudeptrai** [comment]
@DabiaoMa fastspeech1 and fastspeech2  are convergences at around 100->150k. Fastspeech2 is much better.

**Comment by DabiaoMa** [comment]
> @DabiaoMa fastspeech1 and fastspeech2 are convergences at around 100->150k. Fastspeech2 is much better.

Thanks
